doc/design/web.md

```diff
@@ -0,0 +1,153 @@
+# Web Interface design
+
+This design doc will talk about features and web pages needed to let users manage cloud paddle jobs.
+
+## Feature List
+
+- Account Management
+    - Registration, send email to inform if registration succeeded
+    - Account Login/Logout
+    - Password changing, find back
+    - Download SSL keys
+- Jupiter Notebook
+    - Private Jupiter Notebook environment to run Python scripts
+    - Private workspace
+    - Submit job from Jupiter Notebook
+- Job Dashboard
+    - Job history and currently running jobs
+    - Performance Monitoring
+    - Quota Monitoring
+- Datasets
+    - Public Dataset viewing
+    - Upload/Download private datasets
+    - Share datasets
```

Comment:
Maybe need to be more specific: Does it mean can share the dataset to anyone by a link, or just set dataset visible to certain group (similar to the unix read file permission).